index.css

```diff
@@ -0,0 +1,524 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  min-width: 375px;
+
+  .header {
+    width: 100%;
+
+    .social-icons {
+      display: none;
```

Comment:
Please Kindly make use of the right colors for your styling as this is a requirement for this milestone. this makes our project match the given design👍.
<img width="316" alt="image" src="https://github.com/2004-okari/CAPSTONE-01/assets/64643995/13e28945-7368-4f27-8e96-3330bb063827">